doc/examples/example2.md

```diff
@@ -0,0 +1,64 @@
+Certification Authority Example {#example2}
+===========================================
+Certification Authority
+=======================
+
+Chain of Trust 
+--------------
+The example bellow shows how to create a self-signed root certificate with an RSA key
+and use it to sign a new certificate with a CSR
+
+\code{.cpp}
+//Generate an RSA key pair
+AsymmetricKeypair rootRSAKey = AsymmetricKeypair::generateRSA();
+
+//Set the desired values on the distinguished name for the self-signed certificate
+DistinguishedName certDetails = DistinguishedName::Builder{}.organizationalUnitName("Car IT")
```

Comment:
Code formatting is broken in this example. Builder offsets look random.

Reply:
Agree, it's done the correction.